subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/radio/radio.c

```diff
@@ -22,19 +22,26 @@
 #include "hal/ticker.h"
 #include "hal/swi.h"
 
+#define LOG_MODULE_NAME bt_openisa_radio
```

Comment:
Commented about this line in other PRs as well. There should be a line that sets `BT_DBG_ENABLED` based on a kconfig before including common/log.h. otherwise the debug cannot be controlled through Kconfig.

#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER) for example.

Reply:
Right, I've resolved the conflict due to the encryption PR merged and will push soon